source/hardware/controllers/numark_ns6ii.rst

```diff
@@ -0,0 +1,543 @@
+4         :hwlabel:`Deck 3`/:hwlabel:`Deck 4`                                     Switch between controlling deck 1/3 or 2/4
+5         :hwlabel:`USB`                                                          See :ref:`numark-ns6ii-dj-changeover`
+5         :hwlabel:`SHIFT` + :hwlabel:`USB`                                       Switch deck to be controlled by PC1 (white) or PC2 (red)
+6         :hwlabel:`LINE` / :hwlabel:`PC`                                         Select Audio input Source for Mixer Channel 3/4
+15        Crossfader                                                              Fades volume between side A and B
+16        :hwlabel:`Master` Gain                                                  Controls output volume of the hardware Master output
+17        :hwlabel:`Booth` Gain                                                   Controls output volume of the hardware Booth output
+18        Jog Wheel (top surface)                                                 Perform scratch operation (if scratch mode is enabled, otherwise perform as if outer edge was touched)
+18        Jog Wheel (outer edge)                                                  Rotate to lower/raise playback speed (and pitch if key lock is off).
+19        Play/Pause Button                                                       Starts/Stops/Resumes playback on the current Deck
+20        :hwlabel:`CUE`                                                          Specifies, plays or recalls temporary cue point. Depends on :ref:`Cue mode <interface-cue-modes>`
+20        :hwlabel:`SHIFT` + :hwlabel:`CUE`                                       Seeks to the Track start, then stops the Deck.
+21        :hwlabel:`SYNC` button                                                  Short press to match tempo and phase of other deck once. Long press to enable :ref:`Sync Lock <sync-lock>`.
+21        :hwlabel:`SHIFT` + :hwlabel:`SYNC` button                               Press to toggle :term:`quantization`.
+22        :hwlabel:`BLEEP` Button                                                 Press to play in reverse, return to where the track would be with no button press on button release.
+22        :hwlabel:`SHIFT` + :hwlabel:`BLEEP` Button (:hwlabel:`KEYLOCK`)         Toggle :term:`Keylock <key lock>`
+23        :hwlabel:`SCRATCH` button                                               Toggle platter scratching. If active, touching the jogwheel results in scratch behavior, if disabled jogwheel behaves as if only outer edge was touched.
+23        :hwlabel:`SHIFT` + :hwlabel:`SCRATCH` button                            Switch display between showing remaining/elapsed time
+24        :hwlabel:`STRIP SEARCH`                                                 Seek to absolute position in *stopped* track (does nothing for playing tracks to avoid accidental)
```

Comment:
just "accidents"?

Reply:
yeah, didn't know how to finish the sentence and then just forgot. "accidents" is good.